manifests/producer.pp

```diff
@@ -46,6 +46,7 @@
 # }
 #
 class kafka::producer (
+  $input,
```

Comment:
The console producer makes only sense if there is the possibility to read from something